prime-router/src/test/kotlin/ReportTests.kt

```diff
@@ -745,4 +746,21 @@ class ReportTests {
         assertThat(specimenId).isEqualTo("")
         assertThat(observation).isEqualTo("null")
     }
+}
+
+class OptionTests {
+    @Test
+    fun `test valueOfOrNone`() {
+        val option = Options.valueOfOrNone("SkipSend")
+        assertThat(option).equals(Options.SkipSend)
+
+        val deprecatedOption = Options.valueOfOrNone("SkipInvalidItems")
+        assertThat(deprecatedOption).equals(Options.SkipInvalidItems)
+
+        val noneOption = Options.valueOfOrNone("None")
+        assertThat(noneOption).equals(Options.None)
+        val invalidOption = "INVALID OPTION"
+        val msg = "$invalidOption is not a valid Option. Valid options: ${Options.values().joinToString()}"
```

Comment:
I will recommend against checking the text message as that can easily change and makes this test a bit brittle.  Maybe just check there is a message.